sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Artifact/RegistryArtifact.cs

```diff
@@ -315,6 +315,98 @@ Page<ArtifactTagProperties> NextPageFunc(string nextLink, int? pageSizeHint)
             return PageableHelpers.CreateEnumerable(FirstPageFunc, NextPageFunc);
         }
 
+        /// <summary> List the tags that uniquely identify this artifact and the properties of each. </summary>
+        /// <param name="orderBy"> Requested order of tags in the collection. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="RequestFailedException">Thrown when a failure is returned by the Container Registry service.</exception>
+        public virtual AsyncPageable<ArtifactTagProperties> GetTagPropertiesCollectionAsync(ArtifactTagOrder orderBy = ArtifactTagOrder.None, CancellationToken cancellationToken = default)
```

Comment:
Curious: are these part of the blob change, or addressing a different feature requirement?

Reply:
The other APIs are for functionality the Bicep team needed.